developer-docs-site/sidebars.js

```diff
@@ -437,6 +437,39 @@ const sidebars = {
       className: "sidebar-title",
     },
     "nodes/aptos-api-spec",
+    {
+      type: "category",
+      label: "Aptos SDKs",
+      collapsible: true,
+      collapsed: true,
+      link: { type: "doc", id: "sdks/index" },
+      items: [
+        {
+          type: "category",
+          label: "TypeScript SDK",
+          link: { type: "doc", id: "sdks/ts-sdk/index" },
+          collapsible: true,
+          collapsed: true,
+          items: [
+            "sdks/ts-sdk/typescript-sdk-overview",
+            {
+              type: "category",
+              label: "API client layer",
+              link: { type: "doc", id: "sdks/ts-sdk/sdk-client-layer" },
+              collapsible: false,
+              collapsed: false,
```

Comment:
I was hoping for this config to not show the items on the sidebar. Is there a way to include those files without showing them on the sidebar, but when going to the page we can still see the sidebar?
cc @saharct